src/styles/card.css

```diff
@@ -0,0 +1,81 @@
+.card {
+  width: 200px;
+  margin: 0 auto;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: block;
+  width: 100%;
+  height: 134px;
+  margin-bottom: 40px;
+}
```

Comment:
image is bigger than in design, apply these changes to fix it
1) remove width: 100% to preserver aspect ratio
2) center it horizontally with margin: 0 auto

![image](https://user-images.githubusercontent.com/57598825/152980620-04d56dba-6c12-4430-9ac0-0b8188698c10.png)


```suggestion
.card__image {
  display: block;
  height: 134px;
  margin: 0 auto;
  margin-bottom: 40px;
}
```